### PR TITLE
#9461: Fix UI issue in catalog tool using custom metadata template

### PR DESCRIPTION
**Pull request by mahmoudadel54**
## Description
Fix UI issue in catalog tool using a custom metadata template for CSW sources which now the buttons for metadata items in the result list are well positioned.

**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/geosolutions-it/MapStore2/blob/master/CONTRIBUTING.md

**What kind of change does this PR introduce?** (check one with "x", remove the others)
 - [x] Bugfix

## Issue
#9461 
**What is the current behavior?**
Using a custom metadata template for CSW sources introduce a problem in the UI for which buttons for metadata items in the result list are not well positioned.
image

#<issue>
#9461 

**What is the new behavior?**
image

## Breaking change
**Does this PR introduce a breaking change?** (check one with "x", remove the other)
 - [x] No

### CODE REVIEW

File: web/client/themes/default/less/sidegrid.less [hunk 1]
```diff
@@ -290,7 +290,7 @@
                     text-overflow: ellipsis;
                     white-space: unset;
                     overflow: visible;
-                    width: 100%;
+                    // width: 100%;
```
Comment (MV88) [inline web/client/themes/default/less/sidegrid.less:293]: do not comment styles. simply remove it

**Comment by mahmoudadel54** [comment]
> * [ ]  SideCard is used in different part of the application, did you try to check if this change affects them somehow?
>   it could work for the catalog, but you may break it for other components and UI parts.
> * [ ]  Take some time to list here in the PR description where this is used and the result of a proper check by running the application
> * [ ]  commented code or styles should be avoided, there is git history to recover that info if needed

I have checked the code, I found that there is a prop called 'fullText' passed to SideCard Component and based on this prop a class 'full-text' is added. This prop is only passed from RecordItem Component which is in sidebarMenu at the UI issue was. 

The edited style is for:  [ .full-text .msapstore-side-card-info > div span ], which means class 'full-text' must be there. 
So based on that, I think there is no other effect on the other components.

**Review by MV88 — APPROVED** [review]

**Comment by MV88** [comment]
@ElenaGallo Please test it in dev

**Comment by ElenaGallo** [comment]
Test passed, @mahmoudadel54 please backport this to 2023.02.xx. Thanks

**Pull request merged**